Day-II-Calculator/calculator/src/components/DisplayComponents/CalculatorDisplay.js

```diff
@@ -1,2 +1,12 @@
 import React from 'react';
 import './Display.css';
+
+const CalculatorDisplay = (props) => {
+    return(
+        <p>
```

Comment:
Would p or div be a better choice here? I know p is probably a placeholder, but div would be a better choice since this is not and will not be a paragraph.